test/models/host_test.rb

```diff
@@ -2745,10 +2745,10 @@ def to_managed!
   test '#setup_clone is public and clones interfaces so delegated attributes are applied to cloned interfaces' do
     host = FactoryBot.create(:host, :managed)
     original_mac = host.mac
-    host.mac = 'AA:AA:AA:AA:AA:AA'
+    host.mac = 'aa:aa:aa:aa:aa:aa'
```

Comment:
Technically you could even leave this uppercase and verify that it's downcased, right?